src/GravityFormsExtensions.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+namespace P4\MasterTheme;
+
+/**
+ * Class P4\MasterTheme\GravityFormsExtensions
+ * The Gravity form plugin extension, use to add custom functionality like Planet4 form type.
+ */
+class GravityFormsExtensions {
+	/**
+	 * @var string The default gravity form type.
+	 */
+	public const DEFAULT_GF_TYPE = 'other';
+
+	/**
+	 * @var array The Planet4 Gravity form types.
+	 */
+	public const P4_GF_TYPES = [
+		[
+			'label' => 'Other',
+			'value' => 'other',
+		],
+		[
+			'label' => 'Petition',
```

Comment:
Don't we need to translate labels?

Reply:
Not sure if its need to translate 🤔 , cause these values are standard name provided in [Miro board at here](https://miro.com/app/board/uXjVO_vBIYc=/?moveToWidget=3458764522885248018&cot=14). Check with Suzi on JIRA